scripts/pinned-tools.sh

```diff
@@ -1,89 +1,102 @@
 #!/usr/bin/env bash
 error() {
-   local SOURCE_FILE=$1
-   local LINE_NO=$2
-   echo "ERROR: ${SOURCE_FILE}(${LINE_NO}):"
+    local SOURCE_FILE=$1
+    local LINE_NO=$2
+    echo "ERROR: ${SOURCE_FILE}(${LINE_NO}):"
 }
 
 function isLinux() {
-  [[ "$(uname -s)" == "Linux" ]];
+    [[ "$(uname -s)" == "Linux" ]]
 }
 
 function isMacOS() {
-  [[ "$(uname -s)" == "Darwin" ]];
+    [[ "$(uname -s)" == "Darwin" ]]
 }
 
 function isWindows() {
-  ! ( isLinux || isMacOS );
+    ! (isLinux || isMacOS)
 }
 
 function cleanUnity() {
-  rm -rf "${1}/Library"
-  rm -rf "${1}/Temp"
+    rm -rf "${1}/Library"
+    rm -rf "${1}/Temp"
 }
 
-echo "--- Sourcing pinned tools :round_pushpin:"
+function traceStart() {
```

Comment:
Could this somehow reject / strip traces that contain a `;`? To avoid issues

Reply:
ye